test/KisSpec.hs

```diff
@@ -42,17 +44,37 @@ spec = do
                 pat <- req (CreatePatient $ Patient "xy")
                 void $ req (PlacePatient pat (toSqlKey 1)))
             `shouldThrow` (== ConstraintViolation)
-    describe "withSqliteKis" $ do
+    describe "withSqliteKis" $
          it "can run two clients in sequence" $
             withSqliteKis InMemory kisConfig $ \kis -> do
                 patId <- runClient kis $ req (CreatePatient $ Patient "Simon")
                 pats <- runClient kis $ req GetPatients
                 pats `shouldBe` [Entity patId (Patient "Simon")]
+    describe "withSqlitePool" $ do
+          it "can run two clients in sequence" $
+             withTempFile "/tmp/" "tmpKisDB" $ \fp _ ->
+                 withSqliteKis (PoolBackendType (T.pack fp) 10) kisConfig $ \kis ->
+                     do patId <- runClient kis $ req (CreatePatient $ Patient "Simon")
+                        pats <- runClient kis $ req GetPatients
+                        liftIO $ pats `shouldBe` [Entity patId (Patient "Simon")]
+          it "can run two clients in parallel" $
```

Comment:
Der Test funktioniert nicht so richtig, glaube ich.
1. Weiter [oben](https://github.com/lslah/kis-proto/pull/13/files#diff-8b31ab71aba07a0391ff9bf18c22cc46R60) steht `handleRequest = writeNotif req >> runAction req`. D.h. wenn `k_waitForNewNotification kis` returned, muss `runAction` noch nicht gelaufen sein.
2. Die Exceptions in `forkIO` landen nicht im main-Thread, d.h. wenn in den forks irgendetwas schief läuft, schlägt der Test trotzdem nicht fehl.